libs/game/physics.ts

```diff
@@ -373,215 +370,215 @@ class ArcadePhysicsEngine extends PhysicsEngine {
             s._lastY
         );
 
-        const overlappedTiles: tiles.Location[] = [];
```

Comment:
We were making this early in the step for dx and populating it with tiles we 'ran into', but this is unnecessary as the portion at the bottom already checks the tiles the sprite is touching & we're only ever moving < 2px per step here anyways. (and the dy one was actually just keeping it in the scope of the dy movement, and not doing anything with the array)